.github/workflows/multi-arch_images_build.yml

```diff
@@ -0,0 +1,78 @@
+name: Multiarch build
+
+on:
+  push:
```

Comment:
Please, remove this. Including a Github Action that requires setting QUAY USERNAME / PASSWORD is not a good idea. Keep this Github Action to be used manually only

Reply:
For setting  QUAY USERNAME / PASSWORD  this is for pushing multi-arch images on repository where we are keeping those images. So this is must to provide credential to push images on repo.